### PR TITLE
No change in value guide

### PR DESCRIPTION
**Pull request by sarina-dd**
### What does this PR do?
Adds the former Monitor no change in value FAQ to the guide section

### Motivation
Jira request

### Preview
https://docs-staging.datadoghq.com/sarina/no-change-guide/monitors/guides

---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

### CODE REVIEW

File: content/en/monitors/guide/how-can-i-configure-a-metric-monitor-to-alert-on-no-change-in-value.md [hunk 1]
```diff
@@ -1,5 +1,5 @@
 ---
-title: How can I configure a metric monitor to alert on NO change in value?
+title: Configure a metric monitor to alert on NO change in value
 kind: faq
```
Comment (ruthnaebeck) [inline content/en/monitors/guide/how-can-i-configure-a-metric-monitor-to-alert-on-no-change-in-value.md:3]:
```suggestion
kind: guide
```

**Review by ruthnaebeck — CHANGES_REQUESTED** [review]
Also:

- Add an alias pointing to the FAQ page
- Check if the FAQ page link needs to be removed from the FAQ index page
- Check if the FAQ page link is anywhere else in the docs

**Comment by sarina-dd** [comment]
@ruthnaebeck Hey there! I realized after starting to reorg it'd probably be best to just reference in both the FAQ and guide indexes. That way we don't have to mess with aliasing/types/etc. Nils just wants it to appear in guides so it's visible to users / so people know where to find it as FAQ isn't easily accessible.

File: content/en/monitors/guide/_index.md [hunk 2]
```diff
@@ -12,5 +12,6 @@ disable_sidebar: true
     {{< nextlink href="monitors/guide/template-variable-evaluation" >}}Template Variable Evaluation{{< /nextlink >}}
     {{< nextlink href="monitors/guide/monitor-for-value-within-a-range" >}}Monitoring Ranges{{< /nextlink >}}
     {{< nextlink href="monitors/guide/supress-alert-with-downtimes" >}}Downtimes common use cases{{< /nextlink >}}
+    {{< nextlink href="monitors/guide/how-can-i-configure-a-metric-monitor-to-alert-on-no-change-in-value" >}}Configure a metric monitor to alert on NO change in value{{< /nextlink >}}
```
Comment (sarina-dd) [inline content/en/monitors/guide/_index.md:15]:
```suggestion
    {{< nextlink href="monitors/guide/alert-on-no-change-in-value" >}}Configure a metric monitor to alert on NO change in value{{< /nextlink >}}
```

**Comment by sarina-dd** [comment]
@ruthnaebeck Updated! It looks like this wasn't previously mentioned in any of the docs, so should be an easy transition to guide.

File: content/en/monitors/guide/alert-on-no-change-in-value.md [hunk 3]
```diff
@@ -1,6 +1,9 @@
 ---
-title: How can I configure a metric monitor to alert on NO change in value?
-kind: faq
+title: Configure a metric monitor to alert on NO change in value
+kind: guide
+aliases:
+  - monitors/faq/how-can-i-configure-a-metric-monitor-to-alert-on-no-change-in-value
+  - monitors/guide/how-can-i-configure-a-metric-monitor-to-alert-on-no-change-in-value
```
Comment (ruthnaebeck) [inline content/en/monitors/guide/alert-on-no-change-in-value.md:6]: This line is aliasing itself. Should be removed. Otherwise PR looks good 👍

Comment (sarina-dd) [inline content/en/monitors/guide/alert-on-no-change-in-value.md:6]:
```suggestion
```

Comment (sarina-dd) [reply content/en/monitors/guide/alert-on-no-change-in-value.md]: Good catch, removed!

**Review by ruthnaebeck — APPROVED** [review]

**Pull request merged**